.golangci.yml

```diff
@@ -0,0 +1,61 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 5m
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - typecheck
+    - govet
+    - gosec
+#    - nolintlint
+#    - rowserrcheck
+    - gofmt
+    - golint
+    - goimports
+#    - misspell
+#    - bodyclose
+#    - unconvert
+#    - ineffassign
+    - staticcheck
+#    - interfacer
+#    - scopelint
+    - structcheck
+    - deadcode
+#    - depguard
+#    - dogsled
+#    - errcheck
+#    - funlen
+#    - goconst
+#    - gocritic
+#    - gocyclo
+#    - gosimple
+#    - stylecheck
+    - unused
+#    - varcheck
+#    - unparam
+#    - unconvert
+#    - whitespace
+
+issues:
+  exclude-use-default: true
+  exclude-rules:
+    - path: _test\.go
```

Comment:
What about the basic golint, is it included? It finds lots of errors in thrift-generated files that we suppress (see greg in the makefile)

Reply:
Yes, this only excludes `gomnd`,`dupl` and `interfacer`. These typically give false positives in tests (or they will when they are enabled).

Reply:
Added `golint`.